devtools/dbt_helper.py

```diff
@@ -0,0 +1,267 @@
+"""A basic CLI to autogenerate dbt yml."""
```

Comment:
Here's the script to add a new table to the project. It will generate a file called `dbt/models/{data_source}/{table_name}/schema.yml` that defines the basic schema for a table for dbt and adds a test for row counts per partition. I've added some documentation for usage in `dbt/README.md`.

We should decide if there's any other tests we want to automatically add based on our metadata at this point.